docs/usage/search.md

```diff
@@ -24,6 +24,31 @@ Also, using the `Advanced search` button, it is possible to directly put filters
 
 ![Advanced search](assets/advanced-search.png)
 
+### Extended search to uploaded files content
+
+!!! tip "Enterprise edition"
+
+    Extended search to files content is available under the "Filigran entreprise edition" license.
+
+    [Please read the dedicated page to have all information](../administration/enterprise.md)
+
+It's possible to extend the global search by keywords to the content of documents uploaded to the platform via the Data import tab, or directly linked to an entity via its Data tab.
```

Comment:
```suggestion
It's possible to extend the global search by keywords to the content of documents uploaded in the platform via the Data import tab, or directly linked to an entity via its Data tab.
```

Reply:
Generally it is `uploaded to` but we can use `uploaded on` 
e.g:
That video was `uploaded on` the internet
That video was `uploaded to` the website
...🤔